README.rst

```diff
@@ -19,36 +19,34 @@
 .. |Binder| image:: https://mybinder.org/badge_logo.svg
    :target: https://mybinder.org/v2/gh/sunpy/sunpy/main?filepath=examples
 
-SunPy is an open-source Python library for Solar Physics data analysis and visualization.
-Our homepage `SunPy`_ has more information about the project.
+``sunpy`` is an open-source Python library for Solar Physics data analysis and visualization.
```

Comment:
This sentence should state that `sunpy` is the core library of the SunPy ecosytem.

Don't know why "Solar Physics" is capitalized.

Also, `sunpy` doesn't really provide much in the way of data-analysis tools.  Compare this with the verbiage in recent write-ups: "...the `sunpy` core library, a Python software package which is designed to provide the fundamental tools for accessing, loading and interacting with solar physics data in Python."

Reply:
Fixed hopefully.